src/fontra/client/core/utils.js

```diff
@@ -345,9 +345,10 @@ export function arrayExtend(thisArray, itemsArray) {
 }
 
 export function rgbaToCSS(rgba) {
-  const channels = rgba.map((channel) => Math.round(channel * 255));
-  if (channels[3] == 255) {
-    channels.pop();
+  const channels = rgba.slice(0, 3).map((channel) => Math.round(channel * 255));
+  const alpha = rgba[3];
+  if (alpha !== undefined && 0 <= alpha && alpha < 1) {
+    channels.push(alpha);
   }
   return `${channels.length === 4 ? "rgba" : "rgb"}(${channels.join(",")})`;
```

Comment:
I just read https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#syntax and it says:
```
The rgba() function can also be used to express sRGB colors. This is an alias for rgb() that accepts the same parameters.
```
That means we can simplify to:
```suggestion
  return `rgb(${channels.join(",")})`;
```
Would be a nice cleanup while you're at it.